### ISSUE TITLE
Use a Pod Member's location

### ISSUE THREAD

**Issue by awilkescom** [MLH-Fellowship/prep-project-22.JUL.PREP.3#10]
Source a list of locations from Pod members, and allow the user to choose one of these locations to find out the weather. 

For example, it could be a drop down menu with Cities. 

Pod members can submit any location, it doesn't have to be personal!

**Comment by shehabadel** [comment]
Can I work on this issue?

**Comment by awilkescom** [comment]
@shehabadel of course!

**Comment by shehabadel** [comment]
Going to create a `HashMap` to map every single pod member to a location.
So it's going to be something like this
```
const membersLocation = new Map([
    ['Shehab','Cairo']
])
```

**Comment by shehabadel** [comment]
Now it should look like this

https://user-images.githubusercontent.com/53188087/181128850-ce3f4228-ebd8-4589-9b33-607a342893e8.mp4

Looking good? @awilkescom

**Comment by somadisingh** [comment]
@shehabadel hey have you made a PR for this issue?

**Comment by somadisingh** [comment]
Issue successfully implemented by @shehabadel in issue #60
